packages/components/src/color-palette/index.native.js

```diff
@@ -111,6 +186,10 @@ function ColorPalette( {
 			showsHorizontalScrollIndicator={ false }
 			keyboardShouldPersistTaps="always"
 			disableScrollViewPanResponder
+			onScroll={ ( { nativeEvent } ) =>
```

Comment:
I know that it will not increase the performance but could you move it to the named function as it is in the `onContentSizeChange` case?